app/lib/json_log_formatter.rb

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
In most other cases the magic comment has a blank line afterwards, any idea why this file is different?